src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -865,7 +866,7 @@ internal async Task SearchPackagesAndRefreshUpdateCountAsync(string searchText,
             }
         }
 
-        private async Task<bool> GetUseRecommendedPackagesAsync(PackageLoadContext loadContext, string searchText)
+        private bool GetUseRecommendedPackagesAsync(PackageLoadContext loadContext, string searchText)
```

Comment:
naming: probably doesn't need to be async anymore.

Reply:
Good catch! It should definitely not have the Async suffix anymore.